newsroom/wire/items.py

```diff
@@ -15,6 +16,18 @@ class ItemsResource(BaseItemsResource):
     schema["slugline"] = schema["headline"] = schema["body_html"] = schema["description_html"] = metadata_schema[
         "body_html"
     ].copy()
+    schema["expiry"] = {"type": "datetime", "reaonly": True}
+
+
+class ItemsService(BaseItemsService):
+    def get_expired_items(self, expiry_datetime=None, expiry_days=None, max_results=None, include_children=True):
+        # remove old items based on expiry days config
+        for items in super().get_expired_items(expiry_datetime, expiry_days, max_results, include_children):
+            yield items
+        # remove items with custom expiry
+        lookup = {"expiry": {"$lt": expiry_datetime}}
+        items = list(self.get_from_mongo(req=None, lookup=lookup))
```

Comment:
We should probably split these out into chunks, using `max_results` and `last_id` similar to what is used in the parent `get_expired_items`, and yielding each `page` of items